nautobot_circuit_maintenance/templates/nautobot_circuit_maintenance/circuitimpact.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html' %}
+{% extends 'generic/object_detail.html' %}
```

Comment:
In addition to this change, need to review whether any or all of the contents of `block title` and` `block header` are now redundant, and if they can be removed to simplify the template.

Additionally, `block content` should be updated to only override `block content_left_page` as appropriate.

Same comment on the other template files below.